### PR TITLE
fix(clusterloader2): fix daemonset node counter

### PR DESCRIPTION
**Pull request by wrype**
daemonset in load test count node error when exist taints

#### What this PR does / why we need it:
/kind bug
#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: clusterloader2/pkg/measurement/util/runtimeobjects/replicaswatcher.go [hunk 1]
```diff
@@ -150,7 +154,14 @@ func (n *NodeCounter) shouldRun(obj interface{}) (bool, error) {
 		return false, fmt.Errorf("unexpected type of obj: %v. got %T, want *corev1.Node", obj, obj)
 	}
 	matched, err := podMatchesNodeAffinity(n.affinity, node)
-	return !node.Spec.Unschedulable && matched, err
+	tolerate := true
```
Comment (mborsz) [inline clusterloader2/pkg/measurement/util/runtimeobjects/replicaswatcher.go:157]: Can we add an unit test that covers this change?

Comment (wrype) [reply clusterloader2/pkg/measurement/util/runtimeobjects/replicaswatcher.go:157]: ok, have any suggestions about testcase?

Comment (mborsz) [reply clusterloader2/pkg/measurement/util/runtimeobjects/replicaswatcher.go:157]: Maybe something like: a pod that wants to run on a given node, without toleration it wouldn't be possible (test case 1), but it has toleration it's OK (test case 2)

**Pull request merged**